.editorconfig

```diff
@@ -21,10 +21,14 @@ indent_size = 2
 indent_style = tab
 
 # ReST-Files
-[*.rst]
+[*.{rst,rst.txt}]
 indent_size = 4
 max_line_length = 80
 
+# Markdown-Files
+[*.md]
+max_line_length = 80
+
```

Comment:
These changes should go into the core instead which will be synched to the coding-standards package which should be finally used here too.

Reply:
Ok. Feel free to adapt to your likes.